doc/src/quickstart.md

```diff
@@ -71,49 +86,74 @@ calc.l:
 [\t ]+ ;
 ```
 
-calc.y:
-```
+Roughly speaking, each line after the `%%` line is a regular expression (we use
+the [`regex`](https://crates.io/crates/regex) crate), a space character, and a
+quoted lexeme type name. In other words, if the user gives us input such as `234`
```

Comment:
"In other words" is not really fitting here, since this sentence doesn't rephrase the previous one in other words. Maybe "For example" is better here?